### ISSUE TITLE
Minecraft validation dataset

### ISSUE THREAD

**Issue by Dijkstra14** [xizaoqu/WorldMem#9]
Thanks for your great work! As mentioned in the paper, 400 videos were used for validation on the Minecraft dataset, but I could only find 16 videos in the provided validation set. Could you please advise on where to access the full validation set or clarify this discrepancy?

**Comment by xizaoqu** [comment]
Hi, thanks for your interest. The current provided validation dataset is for fast validation during training. We will release the dataset for evaluation and the data generation pipeline in the future.

**Comment by Dijkstra14** [comment]
Awesome, can’t wait for the release!

**Comment by sy-liang123** [comment]
I would like to ask when the validation dataset will be made publicly available?
